indra/tests/test_reading_scripts.py

```diff
@@ -1,12 +1,29 @@
 from __future__ import absolute_import, print_function, unicode_literals
 from builtins import dict, str
 
-from os import path, mkdir
-from indra.tools.reading.read_pmids import READER_DICT, get_proc_num,\
-    get_mem_total
 import pickle
+import random
+import zlib
+from os import path, mkdir
 from nose import SkipTest
 
+from indra.tools.reading.read_db import _convert_id_entry, \
+    get_content_query, get_clauses, upload_readings, \
+    make_db_readings, get_db_readings, _enrich_reading_data,\
+    produce_statements, produce_readings
+from indra.tools.reading.read_files import read_files
+from indra.tools.reading.readers import SparserReader
+from indra.tools.reading.readers import get_readers as get_all_readers
+
+from indra.db import formats
+from indra.tests.test_db import get_db as get_test_db
```

Comment:
I think we should try to import from test_db directly (instead of indra.tests.test_db) and get rid of tests/__init__.py

Reply:
done